driver-service/build.gradle

```
@@ -52,6 +52,7 @@ dependencies {
     implementation 'org.liquibase:liquibase-core'
     implementation("io.micrometer:micrometer-tracing-bridge-brave")
     implementation("io.zipkin.reporter2:zipkin-reporter-brave")
+    implementation "net.ttddyy.observation:datasource-micrometer-spring-boot:${versions.micrometerdatasource}"
```

```suggestion
    implementation ("net.ttddyy.observation:datasource-micrometer-spring-boot:${versions.micrometerdatasource}")
```
ну или другие поменять, чтоб было везде одинаково